### PR TITLE
(#4225) - 4.0.1 blog post , version updates

### PR DESCRIPTION
**Pull request by nolanlawson**
Let me know if I missed anything. This is a rather long
blog post, but I rather like it.

### CODE REVIEW

**Comment by nolanlawson** [comment]
Here is a built version so you don't have to build it yourself: https://s3.amazonaws.com/nolanlawson/pouchdb/www/20150901/index.html

File: docs/_posts/2015-09-01-pouchdb-4.0.1-gotta-go-fast.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+---
+
+I'm a big fan of classic video games. So when I hack on PouchDB, my spirit animal is this guy:
+
+{% include img.html src="sonic_waiting.gif" alt="Sonic the Hedgehog tapping his foot with visible irritation" %}
+
+In *Sonic the Hedgehog*, the titular hero taps his foot impatiently any time you stop moving. It's almost as if he's egging you on, urging you to go faster. *Come on, what's the holdup?*
+
+With PouchDB, performance is one of our top concerns; we don't want any holdups either. So 4.0.1 is a minor release containing some bugfixes, but mostly a lot of speedups.
```
Comment (nolanlawson) [inline docs/_posts/2015-09-01-pouchdb-4.0.1-gotta-go-fast.md]: this should say "patch release"; typo on my part, and I will fix

**Comment by daleharvey** [comment]
:+1: I have been meaning to do more community / informative based release posts, this is awesome

**Comment by nolanlawson** [comment]
Made some small copyedits. And thanks a lot @daleharvey; it means a lot to me that you liked the post. :)

**Comment by nickcolley** [comment]
Excellent post! Love the community news section and performance. Thanks a lot!

**Comment by nolanlawson** [comment]
101e0e8b8300b56fc22cd2de6d3701753590a154

**Pull request closed without merging**